Dockerfile

```
@@ -46,6 +46,8 @@ RUN echo "APT::Install-Recommends \"0\";" >> /etc/apt/apt.conf.d/02recommends &&
 # Install requirements
 WORKDIR /app
 ADD pyproject.toml poetry.lock /app/
+COPY vendor/ /app/vendor/
```

suggestion: let's be modern and try adding `--link`

```suggestion
COPY --link vendor/ /app/vendor/
```

This allows layer caches to be reused in more situations, with the only cost being incompatibility with symlinks as `--link=true` means the command cannot read any files from the previous state, but it is very rare for us to be copying symlinks and so we should be including `--link` by default

Also see https://docs.docker.com/reference/dockerfile/#benefits-of-using---link

---

Ooh this is exciting!

---

Done! I'm just doing a local build to confirm it's happy then I'll push!
Thanks!